app/controllers/uploads_controller.rb

```diff
@@ -9,42 +9,48 @@ def new
     @upload = Upload.from_csv_type(params[:csv_type])
 
     if @upload.csv_type_check?
-      @requirements = requirements_messages
+      csv_requirements
+
       return
     end
-
     alert_and_log(@upload.errors.full_messages.join(', '))
     redirect_to dashboards_path
   end
 
+  def csv_requirements
```

Comment:
can you move this method down below the `private` ?

Reply:
done